apps/src/blockly/addons/cdoXml.js

```diff
@@ -56,6 +56,7 @@ export default function initializeBlocklyXml(blocklyWrapper) {
     //  the rendered blocks and the coordinates in an array so that we can
     //  position them.
     partitionedBlockElements.forEach(xmlChild => {
+      addMutationToMinitoolboxBlocks(xmlChild);
```

Comment:
nit: `miniToolbox`?

Reply:
Yeah, that's fair. I did this intentionally to sort of group "minitoolbox" into a one word label to describe the blocks, but that's probably needlessly convoluted.